open_prices/proofs/ml.py

```diff
@@ -558,6 +558,33 @@ def create_price_tags_from_proof_prediction(
     return created
 
 
+def create_receipt_items_from_proof_prediction(
+    proof: Proof, proof_prediction: ProofPrediction
+) -> list[ReceiptItem]:
+    """Create receipt items from a proof prediction containing receipt item
+    detections."""
+
+    if proof_prediction.model_name != common_google.GEMINI_MODEL_NAME:
+        logger.error(
+            "Proof prediction model %s is not a receipt extraction",
+            proof_prediction.model_name,
+        )
+        return []
+
+    created = []
+    for predicted_item in proof_prediction.data["items"]:
+        receipt_item = ReceiptItem.objects.create(
+            proof=proof,
+            proof_prediction=proof_prediction,
+            price=None,
+            order=len(created) + 1,
```

Comment:
rather use the index of the for loop (seems more logical).
can use `enumerate` to get it :)